src/HierarchicalComboBox/HierarchicalComboBox.js

```diff
@@ -37,6 +39,7 @@ const HierarchicalComboBox = ({
         <Tooltip arrow title={renderInputToolType}>
           <TextField
             {...params}
+            data-cy="scenario-input"
```

Comment:
Could be interesting to prefix the id by  `hierarchical-combo-box` in order to narrow the possibilities to have several field with the same data-cy id?
The scenario information is not relevant here as the hierarchical combobox is agnostic.
Something like  `hierarchical-combo-box-input`